Messenger/AppDelegate.mm

```diff
@@ -135,10 +139,31 @@ - (void)applicationDidFinishLaunching:(NSNotification*)notification {
   _lastNotificationCount = @"";
 }
 
+- (id)callJS:(NSString*)methodName, ... NS_REQUIRES_NIL_TERMINATION {
```

Comment:
Since all arguments are strings and constant, this can be simplified and made more universal as `evaluateJSWithFormat:(NSString*)format, ...` and `evaluateJS:(NSString*)code` used as e.g. `[evaluateJS:@"setActiveConversation(%@)", index]` and `[evaluateJS:@"nextConversation()"]`, respectively.